python/ray/WebUI.ipynb

```diff
@@ -11,16 +11,528 @@
    "cell_type": "code",
    "execution_count": null,
    "metadata": {
-    "collapsed": false
+    "collapsed": true
    },
    "outputs": [],
    "source": [
+    "import binascii\n",
+    "import json\n",
     "import os\n",
+    "import pandas as pd\n",
+    "import pandas.io.sql as psql\n",
+    "import qgrid \n",
     "import ray\n",
+    "import redis\n",
+    "import sys\n",
     "\n",
+    "from bokeh.plotting import figure, show\n",
+    "from bokeh.layouts import layout, widgetbox, row, gridplot\n",
+    "from bokeh.models import ColumnDataSource, HoverTool, Div, CustomJS, Range1d\n",
+    "from bokeh.models.widgets import Slider, Select, TextInput\n",
+    "from bokeh.io import curdoc, output_notebook, push_notebook\n",
+    "from pandas.io.json import json_normalize"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
     "redis_address = os.environ[\"REDIS_ADDRESS\"]\n",
     "ray.init(redis_address=redis_address)"
    ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "addr, port = ray.worker.global_worker.redis_address.split(\":\")\n",
+    "rc = redis.StrictRedis(host=addr, port=port)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Task and Actor Information "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Remote Function Information \n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "fn_table = ray.global_state.function_table()\n",
+    "fn_list = []\n",
+    "if (len(fn_table) != 0): \n",
+    "    for fn_id in fn_table:\n",
+    "        val = fn_table[fn_id]\n",
+    "        val[\"function_id\"] = fn_id\n",
+    "        fn_list.append(val)\n",
+    "else: \n",
+    "    fn_list.append({\"DriverID\": \"None\", \"Module\": \"None\", \"Function\": \"None\", \"FunctionID\": \"None\"})\n",
+    "qgrid.nbinstall(overwrite = True)\n",
```

Comment:
optional arguments should have no spaces around the `=`, so this should be `overwrite=True`.